test/rubygems/helper.rb

```diff
@@ -105,39 +105,32 @@ def refute_directory_exists(path, msg = nil)
     refute File.directory?(path), msg
   end
 
-  # https://github.com/seattlerb/minitest/blob/21d9e804b63c619f602f3f4ece6c71b48974707a/lib/minitest/assertions.rb#L188
-  def _synchronize
-    yield
-  end
-
   # https://github.com/seattlerb/minitest/blob/21d9e804b63c619f602f3f4ece6c71b48974707a/lib/minitest/assertions.rb#L546
```

Comment:
Should this reference be now removed? Maybe say "Originally copied from minitest" or something like that?

Reply:
Make sense. I did update this comment. Thanks!